bindinfo/bind_test.go

```diff
@@ -365,6 +365,43 @@ func (s *testSuite) TestExplain(c *C) {
 	tk.MustExec("drop global binding for SELECT * from t1,t2 where t1.id = t2.id")
 }
 
+// TestBindingSymbolList tests sql with "?, ?, ?, ?", fixes #13871
+func (s *testSuite) TestBindingSymbolList(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	s.cleanBindingEnv(tk)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int, b int, INDEX ia (a), INDEX ib (b));")
+	tk.MustExec("insert into t value(1, 1);")
+
+	// before binding
+	tk.MustQuery("select a, b, sleep(1) from t where a = 3 limit 1, 100")
```

Comment:
Why `sleep(1)`? Won't it slow down the test?

Reply:
Removed.